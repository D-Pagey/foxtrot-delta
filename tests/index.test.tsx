import { render } from "@testing-library/react";

import Home from "../pages";
import { MenuFixture } from "./fixture";

const props = {
  menuData: MenuFixture,
};

describe("Home page", () => {
  it("should render", () => {
    const { container } = render(<Home {...props} />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
