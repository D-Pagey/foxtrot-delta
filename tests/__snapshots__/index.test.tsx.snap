// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render 1`] = `
<div>
  <main
    class="flex flex-col p-4 items-center"
  >
    <h2
      class="mb-4 text-2xl font-bold text-fdBlue"
    >
      Menu
    </h2>
    <ul
      class="max-w-2xl"
    >
      <li
        class="mb-8"
      >
        <h2
          class="font-bold text-xl mb-4 text-fdBlue"
        >
          Mains
        </h2>
        <div
          class="grid items-center gap-x-4 border-b py-4 px-2 cursor-pointer hover:shadow rounded"
          style="grid-template-columns: 1fr max-content;"
        >
          <p
            class="col-start-1 row-start-1 font-bold"
          >
            Curry
          </p>
          <p
            class="col-start-1 row-start-2"
          />
          <p
            class="col-start-1 row-start-3"
          >
            £10.00
          </p>
          <div
            class="relative flex col-start-2 row-start-1 row-end-4 rounded overflow-hidden"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 100px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
            >
              <img
                alt=""
                data-nimg="fixed"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
              />
              <noscript />
            </span>
          </div>
        </div>
        <div
          class="grid items-center gap-x-4 border-b py-4 px-2 cursor-pointer hover:shadow rounded"
          style="grid-template-columns: 1fr max-content;"
        >
          <p
            class="col-start-1 row-start-1 font-bold"
          >
            Pizza
          </p>
          <p
            class="col-start-1 row-start-2"
          />
          <p
            class="col-start-1 row-start-3"
          >
            £10.00
          </p>
          <div
            class="relative flex col-start-2 row-start-1 row-end-4 rounded overflow-hidden"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 100px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
            >
              <img
                alt=""
                data-nimg="fixed"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
              />
              <noscript />
            </span>
          </div>
        </div>
        <div
          class="grid items-center gap-x-4 border-b py-4 px-2 cursor-pointer hover:shadow rounded"
          style="grid-template-columns: 1fr max-content;"
        >
          <p
            class="col-start-1 row-start-1 font-bold"
          >
            Burger
          </p>
          <p
            class="col-start-1 row-start-2"
          />
          <p
            class="col-start-1 row-start-3"
          >
            £10.00
          </p>
          <div
            class="relative flex col-start-2 row-start-1 row-end-4 rounded overflow-hidden"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 100px; height: 100px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
            >
              <img
                alt=""
                data-nimg="fixed"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
              />
              <noscript />
            </span>
          </div>
        </div>
      </li>
    </ul>
  </main>
</div>
`;
